files/class-vip-filesystem.php

```diff
@@ -385,4 +410,77 @@ public function filter_get_attached_file( $file, $attachment_id ) {
 
 		return $file;
 	}
+
+	/**
+	 * Filter `cron_schedules` output
+	 *
+	 * Add a custom schedule for a 5 minute interval
+	 *
+	 * @param   array   $schedule
+	 *
+	 * @return  mixed
+	 */
+	public function filter_cron_schedules( $schedule ) {
+		$schedule['five_minutes'] = [
+			'interval' => 300,
+			'display' => __( 'Once every 5 minutes' ),
+		];
+
+		return $schedule;
+	}
+
+	public function schedule_update_job() {
+		if ( get_option( self::OPT_ALL_FILESIZE_PROCESSED ) ) {
+			wp_clear_scheduled_hook( self::CRON_EVENT_NAME );
```

Comment:
We'll end up triggering `wp_clear_scheduled_hook` on every pageload after attachments are processed which may not be ideal. We should probably check `wp_next_scheduled` first?

```suggestion
			if ( wp_next_scheduled( self::CRON_EVENT_NAME ) ) {
				wp_clear_scheduled_hook( self::CRON_EVENT_NAME );
			}
```